presentations/en/2019-11-05-Working-in-the-open.md

```diff
@@ -157,30 +256,111 @@ It's still your work, you need to maintain it
 
 Note:
 
+- It should be your source of truth, not a clone of your internal workspace
 - Contributions are welcome but you decide what goes into your work or not
+- It's open source so if it dies in the GC, it may live beyond
 
 --
 
+#### Community
+
 > Warning: A community may coalesce
 
-Manage your community
+Manage it for your own benefit!
 
 Note:
 
+- Stephen Walli - "It's all about software engineering economics"
 - If your work is interesting, people might find it
 - People may submit recommendations and modifications to you
 - Manage your community, you will benefit greatly from it
+- Set expectationsGCcollab
+  - "We don't take contributions at this point but happy to have you submit suggestions"
+  - "We will get back to you within 48 hours"
+  - Set templates
+
+--
+
+Be kind, Be patient, Be inclusive
+
+Note:
+
+- Some people are new at this
+- Welcome people from everywhere
+- Set a Code of Conduct
+  - Should include a link to the Public Servant's Values and Ethics0
+
+--
+
+![oh-my-zsh Contributions](assets/images/oss-zsh.png)
 
 ---
 
 ### Tools
 
-- GCmessage
-  - Instant messages
-- GCcollab
+--
+
+#### Community Management
+
+- [GCmessage (pilot)](https://message.gccollab.ca/home)
+- [Slack](https://slack.com/) to a certain extent (closed communities)
+- [Framavox](https://framavox.org/)
+- [GCcollab](https://gccollab.ca/splash/)
   - Communities of Practice
+
+--
+
+#### Workspace
+
 - GCpedia
-- GitLab/GitHub
-- Twitter
-- Open Resources Exchange
-- Slack to a certain extent (closed communities)
+- [GCcollab Wiki](https://wiki.gccollab.ca/)
+- [Framasoft](https://framasoft.org/)
+- [Google Drive + Related Tools](https://www.google.com/drive/)
```

Comment:
I could see the audience saying "yeah, I'd love that but it's blocked by my department. I am not allowed to use those tools".

To prepare for such questions, I suggest:
1- Policy cover from TBS and raising awareness internally that such dogmatic compliance to other policies may inadvertently affect compliance to broader and more higher up TB policies
2- You are able to use your personnal devices, you do not need to always use your government-provided ones  to do work

Reply:
Good point, I'll add the policies.

As an added context, this is a presentation for the Digital Academy cohort. They have been asked to participate with specific device and permission requirements with the hope to have their home department start to better understand these technologies. We already have seen many for which most of these tools are not available. But the DA will help make the case with their departments, directly with CIOs.